1-mandatory/1-writers.js

```diff
@@ -61,8 +66,20 @@ Exercise 2:
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
+function getDeadWritersInTheirForties(writer) {
+  return writer.age > 40 && writer.age < 49 && writer.alive === false;
```

Comment:
i think >= and <= should be used here, so ages of 40 and 49 are included in the range too